scripts/ios-binaries-upload.sh

```diff
@@ -46,14 +46,14 @@ cp "$ADHOC_EXPORT_PATH/$APP_NAME.ipa" "$IOS_BINARIES_DIR_PATH/$IPA_NAME"
 
 cd "$IOS_BINARIES_DIR_PATH"
 git add "$IPA_NAME"
+git status
 
 if [ "$BUILD_CONTEXT" == "ci" ]; then
-  git config --global user.email "ci@librarysimplified.org" ||
-    fatal "could not configure git"
-  git config --global user.name "Library Simplified CI" ||
-    fatal "could not configure git"
+  git config --global user.email "ci@librarysimplified.org"
```

Comment:
The email we set up yesterday was librarysimplifiedci@nypl.org -- should we be using that one as the email of record?

Reply:
I took this email from @io7m scripts some time ago and forgot about it, but yes it makes sense to swap it for the new one.